app/assets/javascripts/components/QuickSettingsMenu/QuickSettingsMenu.tsx

```diff
@@ -42,7 +42,7 @@ const toggleFocusMode = (enabled: boolean) => {
       document.body.classList.remove('focus-mode');
       setTimeout(() => {
         document.body.classList.remove('disable-focus-mode');
-      }, 315);
+      }, 1255);
```

Comment:
Can we move this into a named const that describes where the number came from? `const cssAnimationDuration`